templates/layout.html

```diff
@@ -99,7 +99,7 @@
     </div>
     <footer class="footer">
       <div class="container">
-        <p class="text-muted">
+        <p>
```

Comment:
Why is this CSS class removed?

Reply:
It greatly improves the contrast of the text in the footer.

![image](https://user-images.githubusercontent.com/12371363/59016459-2a8e6800-8839-11e9-8046-739c40fe7865.png)
![image](https://user-images.githubusercontent.com/12371363/59016483-3a0db100-8839-11e9-9400-6975640a8f8d.png)

becomes

![image](https://user-images.githubusercontent.com/12371363/59016502-472aa000-8839-11e9-9b46-40c8d4a5d033.png)
![image](https://user-images.githubusercontent.com/12371363/59016522-53166200-8839-11e9-9298-aa4417eaf770.png)